tests/test_merge_db_functions.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python3
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2022 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+import sys
+import os
+import sqlite3
+
+from tuna.utils.merge_db import parse_jobline
+from tuna.utils.merge_db import parse_text_fdb_name
+from tuna.utils.merge_db import parse_text_pdb_name
+from tuna.utils.merge_db import load_master_list
+from tuna.utils.merge_db import best_solver
+from tuna.utils.merge_db import target_merge
+from tuna.utils.merge_db import no_job_merge
+from tuna.utils.merge_db import single_job_merge
+from tuna.utils.merge_db import multi_job_merge
+from tuna.utils.merge_db import update_master_list
+from tuna.utils.merge_db import write_merge_results
+from tuna.utils.merge_db import merge_text_file
+from tuna.utils.merge_db import merge_sqlite_pdb
+from tuna.utils.merge_db import merge_sqlite_bin_cache
+from tuna.utils.merge_db import merge_sqlite
+from tuna.utils.merge_db import get_file_list
+from tuna.utils.merge_db import get_sqlite_table
+from tuna.utils.merge_db import get_sqlite_row
+from tuna.helper import prune_cfg_dims
+from tuna.utils.merge_db import get_sqlite_data
+
+this_path = os.path.dirname(__file__)
+
+
+def test_merge_db_functions():
+  test_parse_jobline
+  test_parse_text_fdb_name
+  test_parse_text_pdb_name
+  test_load_master_list
+  test_best_solver
+  test_target_merge
+  test_update_master_list
+  test_write_merge_results
+  test_merge_text_file
+  test_get_sqlite_table
+  test_get_sqlite_row
+  test_get_sqlite_data
+  test_get_file_list
```

Comment:
this is not necessary, pytest will call every function in this script that starts with 'test_<>'. This can be removed

Reply:
ok.